docs/source/docker/configuration/configuration.rst

```diff
@@ -105,13 +105,13 @@ The following are parameters which can be passed to the container:
     min_score: 0.9
 
 
-  # sampling
+  # Selection
   # Choose from ['coreset', 'random'].
   method: 'coreset'
   stopping_condition:
     # Float in [0., 1.] for percentage, int for number of samples, -1 means inactive.
     n_samples: -1    
-    # Float, minimum distance between two images in the sampled dataset, -1. means inactive.           
+    # Float, minimum distance between two images in the selected dataset, -1. means inactive.
```

Comment:
in the selected dataset -> in the dataset you are applying the selection strategy